### PR TITLE
docs: reserves KLIP-55, tablespaces

### PR DESCRIPTION
**Pull request by MichaelDrogalis**
Reserves KLIP-55.

### CODE REVIEW

**Comment by vcrfxia** [comment]
Heya @MichaelDrogalis you'll want to also update the table in the main README for KLIPs in order to reserve your KLIP number: https://github.com/confluentinc/ksql/blob/master/design-proposals/README.md

**Comment by MichaelDrogalis** [comment]
Done! Thanks @vcrfxia!

File: design-proposals/README.md [hunk 1]
```diff
@@ -95,3 +95,4 @@ Next KLIP number: **55**
 | [KLIP-52: BYTES data type support](klip-52-bytes-data-type-support.md)                                                           | Proposal |      0.21.0       |            | [Discussion](https://github.com/confluentinc/ksql/pull/7764) |
 | [KLIP-53: Pull Queries on Streams                                                                                                | Proposal |      0.22.0       |            |                                                              |
 | [KLIP-54: Optimized Range Pull Queries on Tables                                                                                 | Proposal |      0.22.0       |            | [Discussion](https://github.com/confluentinc/ksql/pull/7993) |
+| [KLIP-55: Tablespaces](https://github.com/confluentinc/ksql/blob/af5e34f27627b634a33dc8e5bfa4f64ac75069e6/design-proposals/klip-55-tablespaces.md)                                                                                 | Proposal |      TBD       |            | [Discussion](https://github.com/confluentinc/ksql/pull/8109) |
```
Comment (vcrfxia) [inline design-proposals/README.md:98]: I guess the link to the KLIP and the PR link will both be updated when the KLIP itself is populated? Works for me. (Right now the KLIP link is frozen to the current version which is just the placeholder :) )

**Review by vcrfxia — APPROVED** [review]
LGTM

File: design-proposals/README.md (hunk 1 above)
Comment (MichaelDrogalis) [reply design-proposals/README.md:98]: Yep, just threw in something for the moment. Thanks for the callout!

**Pull request merged**